src/ui/units/wizard/components/Dialogs/DialogShapes/DialogShapes.tsx

```diff
@@ -59,9 +58,13 @@ const DialogShapes: React.FC<Props> = ({
     paletteType,
 }: Props) => {
     const [shapesState, setShapesState] = React.useState<ShapesState>({
+        paletteType,
         selectedValue: null,
         mountedShapes:
-            shapesConfig.mountedShapes && Object.keys(shapesConfig.mountedShapes)
+            shapesConfig.mountedShapes &&
+            Object.keys(shapesConfig.mountedShapes) &&
+            (typeof shapesConfig.paletteType === 'undefined' ||
+                shapesConfig.paletteType === paletteType)
```

Comment:
This won't fix the error for the current saved charts, right?
It is the values stored in mountedShapes that need to be validated, not just the type of config. Because there may be a situation with the correct config type, but an incorrect value in mountedShapes